test/fixtures/users.yml

```diff
@@ -1,31 +1,891 @@
-ohm:
+users_001:
   name: Georg Ohm
   email: gohm@example.com
   password_digest: <%= User.digest('password') %>
   admin: true
 
-watt:
+users_002:
   name: James Watt
   email: jwatt@example.com
   password_digest: <%= User.digest('password') %>
 
-pasteur:
+users_003:
   name: Louis Pasteur
   email: lpasteur@example.com
   password_digest: <%= User.digest('password') %>
 
-palmer:
+users_004:
   name: John Palmer
   email: jpalmer@example.com
   password_digest: <%= User.digest('password') %>
 
-tesla:
-  name: Nikola Tesla
-  email: ntesla@example.com
-  password_digest: <%= User.digest('password') %>
+users_005:
```

Comment:
Mostly for consistency. Since I used that yml dump thing, it pulled all the users from the dev db and labeled them accordingly. Its possible I could have left the users untouched, since I dont think there is really any connection between the equipment/rhizome - but considering that I had trouble with associations that I thought was all in place, I decided not to play with that.